rootfs_linux.go

```diff
@@ -168,6 +168,10 @@ func mountToRootfs(m *configs.Mount, rootfs, mountLabel string) error {
 			if err := syscall.Mount("", dest, "none", uintptr(syscall.MS_PRIVATE), ""); err != nil {
 				return err
 			}
+		} else {
+			if err := syscall.Mount("", dest, "none", uintptr(syscall.MS_SLAVE), ""); err != nil {
```

Comment:
Not yet, there is a separate pull request being worked on for handling all three requests.  

You really want that on a container by container basis.  The problem we have now is the entire host OS is being setup Private, which means no mount points can ever be made SHARED or SLAVE.  

This change is just to make ROOTFS PRIVATE allowing mountpoints outside of ROOTFS to be mounted as PRIVATE.  If this works and does not cause the problem @crosbymichael pulled the previous patch for, we can continue working on allowing users to specify differnet mount propogation rules on a container by container basis.